identity/client/src/pages/login/LoginPage.tsx

```diff
@@ -86,10 +140,19 @@ export const LoginPage: React.FC = () => {
   const { t, Translate } = useTranslate();
   const location = useLocation();
   const license = useLicense();
+  const { enqueueNotification } = useNotifications();
+
   const redirectUrl = getRedirectUrl(location.search);
   const onSuccess = useCallback(() => {
     window.location.href = redirectUrl ?? "/identity/users";
   }, [redirectUrl]);
+  const onFail = useCallback(() => {
+    enqueueNotification({
+      kind: "error",
+      title: "Login failed",
+      subtitle: "That username and password combination is incorrect.",
+    });
+  }, [enqueueNotification]);
```

Comment:
Here it's also inconsistent. We don't show toast notifications for failed login credentials
![Uploading image.png…]()